framework/auth/core.py

```diff
@@ -56,7 +56,6 @@ def string_required(value):
 
 
 def validate_history_item(item):
-    string_required(item.get('institution'))
```

Comment:
Removed because validation is more complicated and occurs on client side.  (institution is only required when there is a list item, otherwise it is not.